### PR TITLE
VBO classification documentation

### PR DESCRIPTION
**Pull request by sabrinatoro**

### CODE REVIEW

**Comment by sabrinatoro** [comment]
@katiermullen this is ready for your final review.

File: docs/ontologymodeling/classification.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+
+<img src="Screenshot 2024-07-30 at 12.52.48 PM.png" width="800">
+
+## Sub-breeds, strains, variety, etc
+Distinguishable sub-breeds, strains, or varieties are also included in VBO and are related to the ontological parent breed using an is_a relation.17–19 For example, 'Chihuahua, Long-Haired (Dog)' (VBO:0200339) and 'Chihuahua, Smooth-Haired (Dog)' (VBO:0200340) are subclasses of 'Chihuahua (Dog)' (VBO:0200338).
+
+<img src="Screenshot 2024-07-30 at 12.49.22 PM.png" width="350">
+
+## DAD-IS Breeds reported in a specific country
+Breeds defined as having been reported in a specific country by National Coordinators in DAD-IS are either
```
Comment (katiermullen) [inline docs/ontologymodeling/classification.md:26]: I am wondering if we should specify that these are "country-nominated Food and Agricultural Organization (FAO) National Coordinators https://www.fao.org/dad-is/national-coordinators/en/" ?

**Review by katiermullen — APPROVED** [review]
@sabrinatoro great work! I approve this PR. Please decide if we need to add more details about the FAO DAD-IS National Coordinators, then this PR can be merged.

File: docs/ontologymodeling/classification.md (hunk 1 above)
Comment (sabrinatoro) [reply docs/ontologymodeling/classification.md:26]: In my mind, we are going to refer back to the DAD-IS section of the documentation. But we might need to update this documentation

**Pull request merged**